packages/modeling/src/measurements/measureArea.js

```diff
@@ -23,12 +25,14 @@ const measureAreaOfPath2 = () => 0
  * @returns {Number} area of the geometry
  */
 const measureAreaOfGeom2 = (geometry) => {
-  if (geometry.area) return geometry.area
+  if (cache.has(geometry)) return cache.get(geometry)
```

Comment:
It may be faster just to call get(), and then test of a value is returned.

WeakMap.prototype.get(key)
- Returns the value associated to the key, or undefined if there is none.